src/qibolab/instruments/qm/controller.py

```diff
@@ -388,9 +388,14 @@ def play(
         options: ExecutionParameters,
         sweepers: list[ParallelSweepers],
     ):
-        if len(sequences) > 1:
-            raise NotImplementedError
-        elif len(sequences) == 0 or len(sequences[0]) == 0:
+        if len(sequences) == 0:
+            return {}
+        elif len(sequences) == 1:
+            sequence = sequences[0]
+        else:
+            sequence, _ = unroll_sequences(sequences, options.relaxation_time)
```

Comment:
Thanks @alecandido, moved to an issue (#1011) so that I can merge this now.

About lifting the call to `unroll_sequences` to the general `Platform`, I believe there was a discussion in #885. At the current state, I agree that it should be okey to lift, but I'd leave for later, possibly after we have another working driver, so that "lifting" makes more sense. At this stage, we could even drop the `readout_map`, as I mentioned above, if it is really useless.